docs/getting-started/sensor-full-disk-access.asciidoc

```diff
@@ -3,23 +3,91 @@
 
 Elastic Endpoint Security requires full disk access to protect you from malware and other cybersecurity threats. Full Disk Access permissions is a new privacy feature introduced in macOS Mojave (10.14) that prevents some applications from accessing your data. This means you need to manually grant permission for Elastic Endpoint Security to access these protected areas of your Mac.
 
-This article describes how to enable full disk access for both the Elastic Agent, required in order to enable Elastic Endpoint Security, and the legacy Endgame sensor.
+This article describes how to enable full disk access for the required security system extensions, the Elastic Endpoint sensor, and the legacy Endgame sensor.
+
+[discrete]
+[[macos-permissions]]
+== macOS permissions
+
+The behavior of the Endgame Sensor and Elastic Endpoint differs based on your macOS version. MDM/JAMF users can pre-approve all full disk access without granting permission to the sensors. However, non-MDM/JAMF users will be prompted or not prompted to enable full disk access for required security files depending on the macOS version and sensor type:
```

Comment:
You flip between capitalizing the Endgame `sensor` and not. `Endgame sensor` is fine.